Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -11,73 +11,91 @@ UNARMED
 
 == Unarmed Strike ==
 
-Damage: 20 + STR mod + DEX mod, minimum 10
+Damage: 20 Ballistic + STR mod + DEX mod, minimum 10
 Range: Adjacent (1m)
 Accuracy Modifier: 0
 
+	Some species may have anatomies that allow for exotic damage types, 
+like Electric or Internal, on their Unarmed Strikes. DM discretion applies.
+
 ==============================
 MELEE WEAPONS
 ==============================
 
 ===== LEVEL 1 REQUIREMENT WEAPONS =====
 
-== Eliazar Plasma Torch $240 ==
+== Eliazar General-Purpose Welding Torch $170 ==
 
-Damage: 30 + 2x Plasma Burn
+Damage: 30 Fire + 2x Burn
 Size: Light
 Range: Adjacent (1m)
 Accuracy Modifier: +1
 STR Requirement:   3
 
-	Plasma. Can quickly repair vehicles with welding materials, seal/cut through 
-moderate doors, and make temporary cover. -40% stealth for being quiet on stealth attacks.
+	A vital tool for structural repairs - good for sealing or cutting  
+metal surfaces or slagging mechanisms like locks. Takes about 1 hour to 
+cut a median person-sized opening through half-inch steel plate. -40 on 

```

Comment:
non-blocking: median or medium?

Reply:
I meant median, but it's possible .... that ..... not everyone ... is ..... a ..... ...... ... statistician 😦 (there I got it out) and average might be clearer.